File: w10/d02/John_Murphy/morning/stringFinder.js
function stringFinder(string, subString){
  var i = 0
  while ( i < string.length) {
    if (subString[0] == string[i]) {
      var y = i
      var x = 0
      var result = 0
      while (x < subString.length) {
        if (string[y] != subString[x]){
          result += 1
        }
        x += 1
        y += 1
      }
      if (result == 0) {
        return i
      }
    }
    i++
  }
  return false
}

File: w10/d02/John_Murphy/morning/string_finder.rb
def string_finder(string, sub_string)
  i = 0
  while i < string.length
    if sub_string[0] == string[i]
      y = i
      x = 0
      result = 0
      while x < sub_string.length
        if string[y] != sub_string[x]
          result += 1
        end
        x += 1
        y += 1
      end
      if result == 0
        return i
      end
    end
    i += 1
  end
  return false
end
